ketchup/options.py

```diff
@@ -16,17 +18,21 @@ def calculate_episodes_from_hours(episodes, episode_duration_mins=24):
 
 # Option 0 in Discord.
 def calc_watch_time_hrs(total_eps, episode_length_mins):
-    overall_watch_time = int(total_eps * episode_length_mins) / 60
+    overall_watch_time = total_eps * episode_length_mins / 60
     return overall_watch_time
 
 
-def calc_days_avail_to_watch(start_date, end_date, days_per_week_limit=None):
+def calc_days_avail_to_watch(start_date, end_date, eps_per_week_limit=None):
     # Calculate the difference between the end_date and the start_date.
     calc_date_diff = end_date - start_date
 
     # Extracts the total number of days from the timedelta "day" object.
     avail_days = calc_date_diff.days
 
+    if eps_per_week_limit:
```

Comment:
Checks to see if "eps_per_week_limit" exists and it calculates the number of weeks to cover the available days, and then the `avail_days` is updated by calculating the calculated weeks by the weekly limit, and then the original value of `avail_days` to ensure that it is not increased beyond the calculated limit.